### PR TITLE
feat(api,ui): improve error and display request id in ui

### PR DESCRIPTION
**Pull request by richardlt**
1. Description
1. Related issues
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#13423.0 ✘  
* Build
  * Build and Package All ✘  
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✘  
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#13424.0 ■   
* Build
  * Build and Package All ✘  
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ■   
  * Lint UI Translate ■   
  * Test CLI ■   
  * Test Engine ■   
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✘

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#13430.0 ✘  
* Build
  * Build and Package All ✘  
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#13432.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✔ 
* Package
  * Docker ✔ 
  * Docs ✔ 
* Integration
  * Docker Compose Tests ✘  
  * Standalone Tests ✘  
  * Standalone tests 1 -   
  * Standalone tests 2 -   
  * Standalone tests 3 -

File: sdk/error.go [hunk 1]
```diff
@@ -937,7 +970,7 @@ func (e *MultiError) Append(err error) {
 			e.Append((*mError)[i])
 		}
 	} else {
-		*e = append(*e, err)
+		*e = append(*e, WithStack(err))
```
Comment (fsamin) [inline sdk/error.go:973]: include the multiError Append in the stack of each function may be annoying

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#13450.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✘  
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✔

File: sdk/error.go (hunk 1 above)
Comment (richardlt) [reply sdk/error.go:978]: I added a filter on func name in the print of the log call stack to ignore it.

**Pull request merged**